### PR TITLE
Add support for allSquash, anonUid and anonGid

### PR DESCRIPTION
**Pull request by sopak**

### CODE REVIEW

**Comment by sopak** [comment]
/assign @jsafrane @wongma7
thanks

**Review by cainzhong — APPROVED** [review]

File: nfs/docs/usage.md [hunk 1]
```diff
@@ -7,6 +7,9 @@ Edit the `provisioner` field in `deploy/kubernetes/class.yaml` to be the provisi
 ### Parameters
 * `gid`: `"none"` or a [supplemental group](http://kubernetes.io/docs/user-guide/security-context/) like `"1001"`. NFS shares will be created with permissions such that pods running with the supplemental group can read & write to the share, but non-root pods without the supplemental group cannot. Pods running as root can read & write to shares regardless of the setting here, unless the `rootSquash` parameter is set true. If set to `"none"`, anybody root or non-root can write to the share. Default (if omitted) `"none"`.
 * `rootSquash`: `"true"` or `"false"`. Whether to squash root users by adding the NFS Ganesha root_id_squash or kernel root_squash option to each export. Default `"false"`.
+* `allSquash`: `"true"` or `"false"`. Whether to all usears should be squashed to anonUID and anonGID by adding the NFS Ganesha all_squash or kernel all_squash option to each export. Default `"false"`.
+* `anonUID`: `"none"` or `"integer"`. When rootSquash or allSquash is set, this uid will be used for  squashed users. If set to `"none"`, default of the server will be used.
```
Comment (wongma7) [inline nfs/docs/usage.md:11]: Will be worth mentioning for anonUID and anonGID, a similar to the gid description:

> Default (if omitted) `"none"`

**Comment by wongma7** [comment]
This looks good to me aside from a documentation nitpick. However, the code for this app was recently moved to https://github.com/kubernetes-sigs/nfs-ganesha-server-and-external-provisioner, so if you want this to be built and released it will be better to recreate the PR there and I will drop an /lgtm there. (This repository is still missing a notice about the migration, my apologies).

File: nfs/docs/usage.md (hunk 1 above)
Comment (sopak) [reply nfs/docs/usage.md]: Was the last change what you requested?

**Comment by nikhita** [comment]
@sopak Thanks for your PR!

As @wongma7 mentioned, we are in the process of archiving this repo and are no longer accepting new PRs. Please see https://github.com/kubernetes/org/issues/1563 for more details. 

We have a tracking issue open to pull in changes from this PR though - https://github.com/kubernetes-sigs/nfs-ganesha-server-and-external-provisioner/issues/5.

Going to close this PR in order to archive this repo. Thanks again!

**Pull request closed without merging**